### PR TITLE
Add actor prerequisite check

### PR DESCRIPTION
**Pull request by Taction**
# Description
Invoke while actor application not ready, dapr returns error message `the state store is not configured to use the actor runtime. Have you set the - name: actorStateStore value: "true" in your state store component file?` which should be waiting for the app channel ready.

## Issue reference

Please reference the issue this PR will close: #5554 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [x] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by ItalyPaleAle — APPROVED** [review]

File: pkg/http/api.go [hunk 1]
```diff
@@ -1408,13 +1412,26 @@ func (a *api) findTargetID(reqCtx *fasthttp.RequestCtx) string {
 	return ""
 }
 
-func (a *api) onCreateActorReminder(reqCtx *fasthttp.RequestCtx) {
-	if a.actor == nil {
-		msg := NewErrorResponse("ERR_ACTOR_RUNTIME_NOT_FOUND", messages.ErrActorRuntimeNotFound)
-		respond(reqCtx, withError(fasthttp.StatusInternalServerError, msg))
-		return
+// actorPrerequisiteWrapper warps the actor handler with the common actor prerequisite check.
```
Comment (DeepanshuA) [inline pkg/http/api.go:1415]: nit: 
```suggestion
// actorPrerequisiteWrapper wraps the actor handler with the common actor prerequisite check.
```

**Review by DeepanshuA — DISMISSED** [review]

File: pkg/http/api.go (hunk 1 above)
Comment (Taction) [reply pkg/http/api.go]: good eye! Done.

**Review by amanbha — DISMISSED** [review]

**Comment by yaron2** [comment]
@Taction please check the double actor activation test failure

**Comment by mukundansundar** [comment]
@Taction The actor reminder e2e tests seem to fail. Can you take a look?

**Comment by Taction** [comment]
@mukundansundar I haven't reproduced this error in my local environment, can you please help me retrigger a /ok-to-test here?

**Comment by mukundansundar** [comment]
> @mukundansundar I haven't reproduced this error in my local environment, can you please help me retrigger a /ok-to-test here?

Seems to be a flaky test?

**Comment by Taction** [comment]
Yes, it seems to be a flaky test. Besides, I checked the test code and found no obvious problems, and it works well in my local environment. It could be an occasional network error, which can be ignored if this problem does not show up again.

**Review by mukundansundar — APPROVED** [review]

**Comment by Taction** [comment]
The latest failed e2e test on windows platform is related to `e2e/service_invocation`, which not related to this pr.

**Comment by daixiang0** [comment]
@Taction please fix conflicts.

**Comment by ItalyPaleAle** [comment]
keep-alive

**Comment by ItalyPaleAle** [comment]
@yaron2 I believe #6972 fixed this issue (and fixed #5554 too)

**Pull request closed without merging**